horreum-web/src/components/CustomTable.tsx

```diff
@@ -138,7 +138,7 @@ function CustomTable<D extends object>({
 
     return (
         <Card>
-            {(title || pagination?.top) && (
+            {(title && pagination?.top) && (
```

Comment:
I don't think this should be an `&&` because there might be situation where you want the pagination without the title
```suggestion
            {(title || pagination?.top) && (
```

Reply:
yes, i need to revert this and fix it correctly, there is an issue where the CardHeader is being displayed without a title or pagination